src/collapse/docs/readme.md

```diff
@@ -7,3 +7,27 @@
   <i class="glyphicon glyphicon-eye-open"></i>
   _(Default: `false`)_ -
   Whether the element should be collapsed or not.
+
```

Comment:
Look at the `uib-tab` documentation for what to put here:  default should be `null`, callback could also be an expression and not a function.